app/src/main/java/com/example/development/sakaiclientandroid/api_models/gradebook/AllGradesPost.java

```diff
@@ -7,22 +7,12 @@
 
 import org.apache.commons.lang3.builder.ToStringBuilder;
 
-public class AllGradesObject {
+public class AllGradesPost {
```

Comment:
@akench Is there a need to rename this class? I personally think `AllGradesPost` sounds a bit ambiguous (sounds more like it's the body of a POST request or something).

Reply:
Oh I see. I thought `AllGradesObject` was fine since it conveys the meaning sufficiently. If the `Post` suffix is auto-generated for announcements, then this is fine since it makes it consistent; I'll do the same for the assignments POJOs.